### ISSUE TITLE
YTDL-CORE Age-restricted videos not working on discord.js music bot.

### ISSUE THREAD

**Issue by Mikeisonvacationwithkal** [fent/node-ytdl-core#981]
I made a simple music discord bot and it can't play age-restricted videos. 

It gives out the 404 error meaning it can't access the age-restricted video, is there any way to fix this?  

All I know is that there is the setting cookie way which I quite don't understand
 how it works. Thanks in advance.
Error in console:
![ijij](https://user-images.githubusercontent.com/44837819/126932714-043a14d4-c5ff-4833-808c-cb659bc957d7.png)

**Comment by TheWilfrido** [comment]
You are not using cookies...                                                                                  ?

**Comment by Mikeisonvacationwithkal** [comment]
> You are not using cookies... ?

I know that I'm not using cookies, it's genuinely confusing for me and I have no clue how it works

**Comment by TheWilfrido** [comment]
https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
![image](https://user-images.githubusercontent.com/67902290/126933730-764d0991-d9ec-40ca-af58-fe6ca0977739.png)
Put that "requestOptions" in ytdl.getInfo(...) and ytdl (...)

**Comment by Mikeisonvacationwithkal** [comment]
> https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
> ![image](https://user-images.githubusercontent.com/67902290/126933730-764d0991-d9ec-40ca-af58-fe6ca0977739.png)
> Put that "requestOptions" in ytdl.getInfo(...) and ytdl (...)

Well now I get this error.
```
UnhandledPromiseRejectionWarning: Error: Sign in to confirm your age
```

**Comment by Nico-ITA** [comment]
I have used cookie in order to fix this error but doesnt work. Also this is a duplicate issue

**Comment by Nico-ITA** [comment]
> 
> 
> > https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
> > ![image](https://user-images.githubusercontent.com/67902290/126933730-764d0991-d9ec-40ca-af58-fe6ca0977739.png)
> > Put that "requestOptions" in ytdl.getInfo(...) and ytdl (...)
> 
> Well now I get this error.
> 
> ```
> UnhandledPromiseRejectionWarning: Error: Sign in to confirm your age
> ```

yes this happened to me too . Try to use cookie from an account that can play restrictede video

**Comment by Mikeisonvacationwithkal** [comment]
> > > https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
> > > ![image](https://user-images.githubusercontent.com/67902290/126933730-764d0991-d9ec-40ca-af58-fe6ca0977739.png)
> > > Put that "requestOptions" in ytdl.getInfo(...) and ytdl (...)
> > 
> > 
> > Well now I get this error.
> > ```
> > UnhandledPromiseRejectionWarning: Error: Sign in to confirm your age
> > ```
> 
> yes this happened to me too . Try to use cookie from an account that can play restrictede video

Found out that this is an EU thing that happens to everyone, I moved it to a server and it sends out the 404 error.

**Comment by Nico-ITA** [comment]
> > > > https://github.com/fent/node-ytdl-core/blob/master/example/cookies.js
> > > > ![image](https://user-images.githubusercontent.com/67902290/126933730-764d0991-d9ec-40ca-af58-fe6ca0977739.png)
> > > > Put that "requestOptions" in ytdl.getInfo(...) and ytdl (...)
> > > 
> > > 
> > > Well now I get this error.
> > > ```
> > > UnhandledPromiseRejectionWarning: Error: Sign in to confirm your age
> > > ```
> > 
> > 
> > yes this happened to me too . Try to use cookie from an account that can play restrictede video
> 
> Found out that this is an EU thing that happens to everyone, I moved it to a server and it sends out the 404 error.

Can u exeplain better becouse now my bot  can play age restricted video with cookie but not video with ecplicit contenents or lyrics
